src/relion/dbmodel/modeltables.py

```diff
@@ -217,6 +201,7 @@ class CTFTable(Table):
     def __init__(self):
         columns = [to_snake_case(c) for c in sqlalchemy.CTF.__table__.columns.keys()]
         prim_key = get_prim_key(sqlalchemy.CTF)
+        # this pattern occurs very often. Turn it into an c,p=inspectify() function?
```

Comment:
`get_prim_key()` also (partially) loops over columns, so we could just do both in one